content/es/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,1092 @@
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+Las instantáneas de volúmenes solo admiten los complementos de volumen CSI fuera del árbol. Para obtener más detalles, consulta [Instantáneas de Volúmenes](/es/docs/concepts/storage/volume-snapshots/). Los complementos de volumen dentro del árbol están obsoletos. Puedes obtener información sobre los complementos de volumen obsoletos en el [FAQ de Complementos de Volumen](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md).
+
+### Crear una PersistentVolumeClaim desde una Instantánea de Volumen {#create-persistent-volume-claim-from-volume-snapshot}
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: restore-pvc
+spec:
+  storageClassName: csi-hostpath-sc
+  dataSource:
+    name: new-snapshot-test
+    kind: VolumeSnapshot
+    apiGroup: snapshot.storage.k8s.io
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+```
+
+## Clonación de Volúmenes
+
+[Clonación de Volúmenes](/es/docs/concepts/storage/volume-pvc-datasource/)
+solo está disponible para complementos de volumen CSI.
+
+### Crear una PersistentVolumeClaim desde una PVC existente {#create-persistent-volume-claim-from-an-existing-pvc}
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cloned-pvc
+spec:
+  storageClassName: my-csi-plugin
+  dataSource:
+    name: existing-src-pvc-name
+    kind: PersistentVolumeClaim
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+```
+
+## Pobladores de Volúmenes y Fuentes de Datos
+
+{{< feature-state for_k8s_version="v1.24" state="beta" >}}
+
+Kubernetes admite pobladores de volúmenes personalizados.
+Para utilizar pobladores de volúmenes personalizados, debes habilitar la característica `AnyVolumeDataSource`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) para el kube-apiserver y kube-controller-manager.
+
+Los pobladores de volúmenes aprovechan un campo de especificación de PVC llamado `dataSourceRef`. A diferencia del campo `dataSource`, que solo puede contener una referencia a otra PersistentVolumeClaim o a un VolumeSnapshot, el campo `dataSourceRef` puede contener una referencia a cualquier objeto en el mismo espacio de nombres, excepto los objetos principales que no sean PVC. Para los clústeres que tienen habilitada la característica, se prefiere el uso de `dataSourceRef` en lugar de `dataSource`.
+
+## Fuentes de Datos entre Espacios de Nombres
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+Kubernetes admite fuentes de datos de volúmenes entre espacios de nombres.
+Para utilizar fuentes de datos de volúmenes entre espacios de nombres, debes habilitar las características `AnyVolumeDataSource` y `CrossNamespaceVolumeDataSource`
+[feature gates](/docs/reference/command-line-tools-reference/feature-gates/) para el kube-apiserver y kube-controller-manager.
```

Comment:
@edithturn from the internal discussion at the weekly meeting, we decided to use:

```suggestion
[Interruptores de funcionalidades (feature gates)](/docs/reference/command-line-tools-reference/feature-gates/) para el kube-apiserver y kube-controller-manager.
```

could you apply this final change, please?